### ISSUE TITLE
Functionality for 'Add additional search fields'

### ISSUE THREAD

**Issue by ErwinKomen** [ErwinKomen/RU-passim#6]
Add functionality to the button `Add additional search fields`.
When the user clicks this, an additional set of search fields should be added.

Question: does this combine with the previous search fields through 'AND' or through 'OR'?

**Comment by ErwinKomen** [comment]
The first place to implement this is in `manuscript.html`.

**Comment by ErwinKomen** [comment]
The basics are implemented, but:
1. Need to add 'delete row' (button + functionality)
1. Need to add the combination functionality (either 'and' or 'or'; probably the latter)

**Comment by ErwinKomen** [comment]
Okay, deletion is now possible and tested. And (2) is done too: they're all 'and'.

**Comment by ErwinKomen** [comment]
TODO: make sure the search criteria are taken into account when searching

**Comment by ErwinKomen** [comment]
The search criteria for the different **or** parts are now taken into account.
Also: I managed to have `typeahead` work correctly over the different rows (where each row is one form in a `formset`). I did this by modifying the `bloodhound` variable definitions in `passim.js`: refer to my own function of `remote.replace`. E.g. `ru.passim.tt_library()`. This function retrieves the correct values for possibly selected `city` and `country`, and incorporates them in the search URL that is used. 
Note: this system makes use of continuously capturing the current `<input>` position using the variable `loc_elInput`, as defined by the event handler in `init_typeahead()`
